public_html/wp-content/mu-plugins/blocks/source/components/item/controller.php

```diff
@@ -29,9 +30,15 @@ function render_item_title( $title, $link = '', $heading_level = 3, array $class
 		$classes
 	) );
 
+	$style = '';
+
+	if ( in_array( $align, [ 'left', 'center', 'right' ], true ) ) {
+		$style = "text-align:$align;";
+	}
+
 	ob_start();
 	?>
-	<<?php echo esc_html( $tag ); ?> class="<?php echo esc_attr( $classes ); ?>">
+	<<?php echo esc_html( $tag ); ?> class="<?php echo esc_attr( $classes ); ?>" style="<?php echo esc_attr( $style ); ?>">
```

Comment:
Could we add a class like `align-center` or `aligncenter` (or something else) instead of adding an inline style? Seems like that's usually considered a best practice and would give more flexibility to organizers who are customizing the CSS...

Reply:
(Same question applies in the **item-title.js** file)

Reply:
I'm following how the core heading block alignment works, and that makes sense to me - if you've explicitly aligned the heading, you don't want some theme styles accidentally overriding it.

> would give more flexibility to organizers who are customizing the CSS

I'm not sure what the use case would be for someone setting "align left" and then wanting to change that to right/center by the theme, where they couldn't just click the right alignment button.

In any case, it is possible to override with `!important` in their CSS (not that I'd encourage that, but it is possible)

Reply:
I wasn't thinking about flexibility as overriding the style so much as being able to select things easily. E.g. `.wordcamp-block__item-title.align-center { ... }`

But, I don't feel too strongly, and following the same behavior as Core's heading block sounds good to me.